stdlib/public/core/UnsafePointer.swift.gyb

```diff
@@ -586,6 +586,22 @@ extension ${Self} {
     fatalError("unavailable function can't be called")
   }
 }
+
+extension Int {
+  @_transparent
```

Comment:
I don't think you need `@_transparent` here -- please drop it unless it breaks something.